### PR TITLE
increased drive size from 500GB to 5T

### LINKED ISSUE #24: Increase 500GB install drive limit
Currently you need a drive under 500GB to install 4.0.2 and this is confusing for new users.  Sometimes the install drive will be bigger (as in my case) because I have spare drives that aren't being used and one is 4TB.  I propose increasing the hard drive limit to 5TB and give users the choice to install there or not (the drives will show up before install overwrites them and this is normal behavior).

I am using this issue to submit a PR fixing this on the boot install line instead of reworking underlying components.  A change to the kiwi file should suffice.

### PR DESCRIPTION
**Pull request by magicalyak**
For [Issue 24](https://github.com/rockstor/rockstor-installer/issues/24) to increase the install drive size for larger drives.  This just modifies the default boot line on the installer using the kiwi file.  It simply adds another 0 to the 500GB line.

Fixes #24

### CODE REVIEW

**Comment by phillxnet** [comment]
@magicalyak I suppose the installer does given a stern warning that all data on the selected drive will be 'destroyed' and then givens a back out option. So with this option we are really only excluding the 6/8/12/ etc drives from showing up.

@FroggyFlox Your thoughts on this. I'm favouring popping it in actually, as it is a bit of a pain but I also quite like the safety feature of simply not offering an install otherwise. But in that case we exclude folks who are not going to be aware of the grub option, even if we add it to the docs. So it may be @magicalyak contribution here is a middle ground approach that at lease exclude the large drives. Still seems a waste for folks to use such large drives as the system disk but as stated by @magicalyak in issue and acknowledge here there is a stern warning shortly there after with accompanying back-out option.

Pretty much means we are allowing install by default on up to 4 GB drives essentially.

**Comment by phillxnet** [comment]
@magicalyak Thanks for this by the way. Much appreciated. Me and @FroggyFlox did discuss this in the early closed days of developing this config but it was one of many items we just had to sort and move on so good to have it under review again via this pull request.
Cheers.

**Comment by FroggyFlox** [comment]
Thanks for the feedback, @magicalyak !

@phillxnet , we knew we would most likely need further tweaking of this option, so I'm not surprised but glad to get feedback about it :)
I agree with you that we should detail that further in our docs, or emphasize it even more than it is now. We will always have users that go through that before seeing the documentation, though, so we need to try and find the option that will give a good user experience in most scenarios. The disks size is already detailed in the dialog where one needs to chose a destination disk, so that should provide enough information for the big majority of cases. That still leaves a doubt in cases in which the user has to choose between two (or more) disks of similar capacity, but that will always be the case, no matter the `oem.disksize` we choose.

In summary, yes, I too am ok with this change :+1: .

Maybe we could also ask in the forum via a poll what most users are using as disk drive to have a better idea of what most of the user base is using; this might provide insight on how to further refine this setting in the future if needed.

**Comment by phillxnet** [comment]
@FroggyFlox Agreed. And we have ourselves a consensus of sorts.

> Maybe we could also ask in the forum via a poll what most users are using as disk drive to have a better idea of what most of the user base is using; this might provide insight on how to further refine this setting in the future if needed.

Yes that's a good idea. And in the interim lets pop in what we have from @magicalyak here and call that our new norm, at least for the time being. Though my preference would be to have it a tad lower but as stated we can see what the wider opinion ends up being via a forum poll or the like and re-visit if need be.

**Comment by phillxnet** [comment]
@magicalyak  Thanks again for stepping up to this one.
I've made a minor edit to the original pr text so we get auto closure of the associated issue: i.e. the "Fixes #24" line.

Happy to merge, as discussed, and thanks for also doing some usability testing here.
